tooling/minimize/src/rvalue.rs

```diff
@@ -180,6 +180,15 @@ impl<'cx, 'tcx> FnCtxt<'cx, 'tcx> {
                         ));
                         ValueExpr::Variant { discriminant, data, enum_ty: ty }
                     }
+                    Type::Ptr(PtrType::Raw { .. }) => {
+                        let ptr = self.translate_operand_smir(&operands[0], span);
```

Comment:
Please add an assertion that `operands.len() == 2`, just to document that we are never omitting an operand here.